torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -1191,20 +1191,18 @@ void setItem(const c10::List<T>& list, int64_t idx, T&& value) {
   list.set(normalized_idx, std::move(value));
 }
 
-template <typename T>
 int listAppend(Stack& stack) {
-  T el = pop(stack).to<T>();
-  c10::List<T> list = pop(stack).to<c10::List<T>>();
+  IValue el = pop(stack).to<IValue>();
+  c10::List<IValue> list = pop(stack).to<c10::List<IValue>>();
```

Comment:
I don't know how `c10::List` works under the hood, but if you have a `c10::List<int64_t>` and try to use it as `c10::List<IValue>` here won't you get some covariant type issues?

Reply:
This is exactly how the code was working previously. There's no functional difference.